### ISSUE TITLE
sb doesn't support dev / build command

### ISSUE THREAD

**Issue by RodrigoTomeES** [storybookjs/storybook#18420]
**Describe the bug**
When using Storybook v.7.0.1 Alpha you need to use sb (#17899) to run storybook but sb doesn't cotains dev / build command.

**To Reproduce**
`npx sb@next upgrade --prerelease`

**System**
```bash
Environment Info:

  System:
    OS: macOS 12.4
    CPU: (8) arm64 Apple M1 Pro
  Binaries:
    Node: 16.15.0 - ~/.nvm/versions/node/v16.15.0/bin/node
    Yarn: 1.22.15 - ~/.nvm/versions/node/v16.15.0/bin/yarn
    npm: 8.5.5 - ~/.nvm/versions/node/v16.15.0/bin/npm
  Browsers:
    Chrome: 102.0.5005.61
    Firefox: 101.0
    Safari: 15.5
  npmPackages:
    @storybook/addon-a11y: ^7.0.0-alpha.1 => 7.0.0-alpha.1 
    @storybook/addon-actions: ^7.0.0-alpha.1 => 7.0.0-alpha.1 
    @storybook/addon-docs: ^7.0.0-alpha.1 => 7.0.0-alpha.1 
    @storybook/addon-essentials: ^7.0.0-alpha.1 => 7.0.0-alpha.1 
    @storybook/addon-links: ^7.0.0-alpha.1 => 7.0.0-alpha.1 
    @storybook/addon-postcss: ^2.0.0 => 2.0.0 
    @storybook/builder-webpack5: ^7.0.0-alpha.1 => 7.0.0-alpha.1 
    @storybook/manager-webpack5: ^7.0.0-alpha.1 => 7.0.0-alpha.1 
    @storybook/react: ^7.0.0-alpha.1 => 7.0.0-alpha.1 
```

<img width="883" alt="imagen" src="https://user-images.githubusercontent.com/13077343/172248127-c7ba5df2-1eb2-4303-8235-e736ee74d222.png">

<img width="330" alt="imagen" src="https://user-images.githubusercontent.com/13077343/172248214-11fbae4d-a4ba-4ed8-8351-229662642225.png">

**Comment by shilman** [comment]
You need to use `sb@7.0.1` also

**Comment by shilman** [comment]
@RodrigoTomeES sorry for the hassle -- we'll be fixing this all with automigrations etc. coming soon

**Comment by RodrigoTomeES** [comment]
@shilman great thanks!

**Comment by unicornware** [comment]
having the same issue using version `7.0.0-alpha.12`.

`sb@7.0.1` doesn't seem to exist:

```
➤ YN0000: ┌ Resolution step
➤ YN0001: │ Error: sb@npm:7.0.1: No candidates found
    at le (/Users/lex/Projects/FLDV/FLDV-P017/.yarn/releases/yarn-3.2.1.cjs:442:7864)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async Promise.allSettled (index 122)
    at async uo (/Users/lex/Projects/FLDV/FLDV-P017/.yarn/releases/yarn-3.2.1.cjs:395:10390)
➤ YN0000: └ Completed in 0s 592ms
➤ YN0000: Failed with errors in 0s 600ms
```

installing `sb@7.0.0-alpha.12` doesn't work either.

how can i run storybook 7? is waiting on new auto migrations required?

**Comment by shilman** [comment]
Typo, sorry. Fixed

**Comment by unicornware** [comment]
no problem and thanks for the quick response! i ended up installing `@storybook/cli@7.0.0-alpha.12` and i just realized my actual issue is running `sb dev` from the command line, rather than via yarn (`sb dev --ci --port 3001` vs `yarn dev`).

would you happen to know if that's an issue on my end or just something to do with how the cli is implemented? @shilman

**Comment by shilman** [comment]
We’re cleaning house! Storybook has changed a lot since this issue was created and we don’t know if it’s still valid. Please open a new issue referencing this one if:
- this is still a problem in SB 7.x
- you can provide a consistent reproduction in 7.x (at https://storybook.new/ or per [repro docs](https://storybook.js.org/docs/react/contribute/how-to-reproduce)).
